test/bin/ci_phase_iso_build.sh

```
@@ -205,7 +209,15 @@ cd "${ROOTDIR}"
 # Get firewalld and repos in place. Use scripts to get the right repos
 # for each branch.
 $(dry_run) bash -x ./scripts/devenv-builder/configure-vm.sh --skip-dnf-update --no-build --force-firewall "${PULL_SECRET}"
-$(dry_run) bash -x ./test/bin/manage_composer_config.sh create
+
+# RPM builds skip composer entirely — install required packages and start the web server.
+if ${RPM_BUILDS} ; then
+    $(dry_run) sudo dnf install -y "https://dl.fedoraproject.org/pub/epel/epel-release-latest-$(rpm -E %{rhel}).noarch.rpm" || true
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)